semantic/semantic.fbs

```diff
@@ -267,11 +252,6 @@ table IdentifierExpression   {
     name:string;
 }
 
-table Identifier {
-    loc:SourceLocation;
-    name:string;
-}
-
```

Comment:
Why was `Identifier` removed? `Identifier` and `IdentifierExpression` are distinct concepts. For instance, `IdentifierExpression` is an actual expression and therefore has a type associated with it. The `Identifier` associated with an object property or a function parameter however does not.

Reply:
@jlapacik Yes, you are right.  I didn't catch this when I reviewed.  I'll make another PR that fixes this.